mod/storage/pkg/filedb/range_db.go

```diff
@@ -94,7 +94,8 @@ func (db *RangeDB) DeleteRange(from, to uint64) error {
 		return errors.New("rangedb: delete range not supported for this db")
 	}
 	for ; from < to; from++ {
-		if err := f.fs.RemoveAll(fmt.Sprintf("%d/", from)); err != nil {
+		path := strconv.FormatUint(from, 10) + "/"
```

Comment:
Curious : What performance benefit do we get.

Reply:
its within the realm of ns but several multipliers faster